app_feup/lib/view/Widgets/canteen_card.dart

```diff
@@ -0,0 +1,83 @@
+
+import 'package:uni/model/app_state.dart';
+import 'package:flutter/material.dart';
+import 'package:tuple/tuple.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:uni/view/Widgets/canteen_row.dart';
+import 'package:uni/view/Widgets/request_dependent_widget_builder.dart';
+import 'package:uni/view/Widgets/row_container.dart';
+import 'date_rectangle.dart';
+import 'generic_card.dart';
+
+class CanteenCard extends GenericCard {
```

Comment:
To make this compliant with #416, we should call this `RestaurantCard` (the same pattern applies elsewhere).